v2/prixite/pages/careers.js

```diff
@@ -0,0 +1,36 @@
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import Head from "next/head";
+import Header from "../components/header";
+import careersPageBg from "../public/images/careersPageBg.jpg";
+
+export default function ContactUs() {
```

Comment:
Name of every component should match the name of the file. So you should create another file called "ContactUs.js" in v2/prixite/pages and import "ContactUs" from that file.

Reply:
Plus I think the name of the component should be `Careers`?

Reply:
Got it